sematic/api/endpoints/resolutions.py

```diff
@@ -261,3 +262,22 @@ def cancel_resolution_endpoint(
     )
 
     return flask.jsonify(dict(content=resolution.to_json_encodable()))
+
+
+# Allow getting a list of external resource ids.
+# Q: Why not just return the actual resources?
+# A: Because we should have the endpoint for getting an external resource
+#    hooked up such that it actually interacts with the external compute
+#    to get the most up-to-date status of the records. It would be too much work
+#    for one service call to have to do that for a *list* of resources.
```

Comment:
What is the likelihood of there being more than one external resource?

Other option, why not decoupling the getting of the external resource record with the updating of said resource with actual external compute?

We could have
`GET /api/v1/resolutions/<id>/external_resources` return a dumb list of external resources (most likely one or two), and then
`GET /api/v1/external_resources/<id>?refresh=true` to get the most up-to-date data from the external resource.

Reply:
Since you're getting by thew root id, the likelihood is actually non-trivial. Sure, I'll add `refresh` as a parameter.